src/routes/+layout.server.ts

```diff
@@ -59,24 +65,52 @@ export const load: LayoutServerLoad = async ({ locals, depends }) => {
 
 	const loginRequired = requiresUser && !locals.user && userHasExceededMessages;
 
+	const disableAssistants = !(ENABLE_ASSISTANTS === "true");
```

Comment:
```suggestion
	const disableAssistants = ENABLE_ASSISTANTS !== "true";
```

was there reason not to use like this?

Reply:
boolean logic can be tricky sometimes depending on the truthiness of different types. So I thought maybe there might be some catch 🤔

Reply:
i think we just went back and forth on whether the flag should enable or disable assistants so I just put a `!` in front of the whole thing without thinking 😆